### ISSUE TITLE
Secure portainer by default

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-base/RADAR-Docker#135]
Portainer is badly secured in the default configuration. We should specify the default admin password in `.env` and use it in the portainer container as described here:
https://portainer.readthedocs.io/en/latest/configuration.html#admin-password

**Comment by blootsvoets** [comment]
We should probably also limit the IP addresses of people that can connect to the portainer path, e.g.
```nginx
location /portainer {
    allow 1.2.3.0/24; # Subnet for KCL?
    allow 91.206.81.2; # The Hyve
    deny all;
    ...
}
```

**Comment by yatharthranjan** [comment]
closed by #149

**Comment by afolarin** [comment]
deploying netdata in a container, and securing. 
https://docs.netdata.cloud/packaging/docker/
